appveyor.yml

```diff
@@ -0,0 +1,22 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: c:\projects\symfony
+
+init:
+  - SET PATH=c:\tools\php\;%PATH%
+
+install:
+  - cinst gow
+  - cinst php
+  - cd c:\tools\php
+  - php -i
+  - ps: Start-FileDownload 'https://getcomposer.org/composer.phar'
+  - echo @php "%~dp0composer.phar" %*>composer.bat
```

Comment:
I'm not sure you are echoing the right thing to the bat file here. I think you need quotes around the echoed content